sql/driver/driver_test.go

```diff
@@ -507,6 +506,59 @@ func TestSelectNoTable(t *testing.T) {
 		}
 	}
 }
+func TestSelectParameters(t *testing.T) {
```

Comment:
i think this is a lot of boilerplate - how about just adding the two important lines to the existing select test?